bin/init.js

```diff
@@ -30,56 +33,284 @@ const info = {
+		return;
+	}
+
+	const composerJson = require( composerJsonPath );
+
+	try {
+		// Remove scripts.
+		delete composerJson.scripts['post-install-cmd'];
+
+		// Commit the changes to file.
+		fs.writeFileSync( composerJsonPath, JSON.stringify( composerJson, null, 2 ) );
+		console.log( info.success( '\ncomposer.json updated successfully!' ), '✨' );
+	} catch ( error ) {
+		console.log( info.error( `Error while updating composer.json: ${ error.message }` ) );
+	}
+}
+
+/**
+ * Modify package.json file.
+ *
+ * @return {void}
+ */
+const modifyPackageJson = () => {
+	console.log( info.message( '\nRemoving init script from the package.json...' ) );
+	const packageJsonPath = path.resolve( getRoot(), 'package.json' );
+
+	if ( ! fs.existsSync( packageJsonPath ) ) {
+		return;
+	}
+
+	const packageJson = require( packageJsonPath );
+
+	try {
+		delete packageJson.scripts['init'];
+
+		if ( ! packageJson.scripts['prepare'] ) {
+			return;
+		}
+
+		const prepareScript = packageJson.scripts['prepare'];
+
+		// Check if 'npm run init' is part of the prepare script.
+		if ( ! prepareScript.includes('npm run init') ) {
+			return;
+		}
+
+		// Split the prepare script into an array of individual scripts.
+		const prepareScriptArray = prepareScript.split( '&&' ).map( ( script ) => script.trim() );
+
+		// Find the index of 'npm run init' in the array.
+		const initScriptIndex = prepareScriptArray.indexOf( 'npm run init' );
+
+		// Remove 'npm run init' from the array if it exists.
+		if ( -1 !== initScriptIndex ) {
+			prepareScriptArray.splice( initScriptIndex, 1 );
+		}
+		// Join the array back into a string and update the 'prepare' script in packageJson.
+		packageJson.scripts['prepare'] = prepareScriptArray.join( ' && ' );
+
+		// Commit the changes to file.
+		fs.writeFileSync( packageJsonPath, JSON.stringify( packageJson, null, 2 ) );
+		console.log( info.success( '\npackage.json updated successfully!' ), '✨' );
+	} catch ( error ) {
+		console.log( info.error( `Error while updating package.json: ${ error.message }` ) );
+	}
+}
+
+/**
+ * Ask theme claenup question.
+ *
+ * @return {void}
+ */
+function themeCleanupQuestion() {
 	rl.question( 'Would you like to run the theme cleanup? (y/n) ', ( cleanup ) => {
 		if ( 'n' === cleanup.toLowerCase() ) {
 			console.log( info.warning( '\nExiting without running theme cleanup.\n' ) );
-			process.exit( 0 );
+		} else {
+			runThemeCleanup();
 		}
-		runThemeCleanup();
 		rl.close();
 	} );
-} else {
-	console.log( info.error( '\nInvalid arguments.\n' ) );
-	process.exit( 0 );
 }
-rl.on( 'close', () => {
-	process.exit( 0 );
-} );
+
+/**
+ * Initialize Git.
+ *
+ * @return {void}
+ */
+const initializeGit = async () => {
+	// Initialize git.
+	console.log( info.success( '\nInitializing git...' ) );
+
+	// Check if .git file exists.
+	const gitDir = path.resolve( getRoot(), '.git' );
+	if ( fs.existsSync( gitDir ) ) {
+		// Remove .git directory.
+		fs.rmSync( gitDir, {
+			recursive: true,
+		} );
+	}
```

Comment:
should we handle the errors here as well with a try..catch block? else there can be a halt in the script in case it is not able to delete the `.git` folder due to some issue like permissions.